app/src/main/java/com/duckduckgo/app/browser/BrowserWebViewClient.kt

```diff
@@ -88,7 +89,7 @@ class BrowserWebViewClient @Inject constructor(private val requestRewriter: Duck
     /**
      * Access the webview url from another thread, jumps onto the main thread to achieve this
```

Comment:
Optional: Update the comment to match? e.g change it to `from another thread` to `from any thread`